packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -4,8 +4,10 @@
         :class="[
             $style['o-btn'],
             $style[`o-btn--${buttonType}`],
-            $style[`o-btn--size${buttonSizeClassname}`],
-            (isFullWidth ? $style['o-btn--fullWidth'] : '')
+            (isIcon ? $style[`o-icon-size${buttonSizeClassname}`] :
+                $style[`o-btn--size${buttonSizeClassname}`]),
```

Comment:
I would leave this as it was, and just use the modifier to change the width of the btn 

```suggestion
            $style[`o-btn--size${buttonSizeClassname}`],
```